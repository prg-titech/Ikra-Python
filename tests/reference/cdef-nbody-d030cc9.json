{
    "basename": "cdef-nbody-d030cc9",
    "cmd": "new-src/interface.py --emit-cdef {infile}",
    "infile": "examples/nbody/nbody.py",
    "infile_hash": "6f95c9faf5c3a5e37fabf40b92e892f22609d9e9068a718d59dedde4",
    "stdout": "cdef-nbody-d030cc9.stdout",
    "stdout_hash": "db0c5c12e550ca3a0f16c53bfd1f197578fa84a843bcfd2d78e567d3",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}