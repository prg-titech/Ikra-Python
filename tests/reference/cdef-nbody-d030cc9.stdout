
int Body_Body_compute_force();
int Body_Body_update();
int parallel_new_Body(int object_num);
int AllocatorInitialize();
int Body_do_all(void (*pf)(int, float, float, float, float, float, float, float));
int AllocatorUninitialize();
